api/AnimationEvent.json

```diff
@@ -0,0 +1,280 @@
+{
+  "api": {
+    "AnimationEvent": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/AnimationEvent",
+        "support": {
+          "webview_android": {
+            "version_added": null,
+            "prefix": "webkit"
+          },
+          "chrome": [
+            {
+              "version_added": "43"
```

Comment:
Is this how we note when something was unprefixed? @Elchi3

Reply:
Yes, the array indicates support ranges. 
So, there is a support range starting from version 43 where this is fully available and unprefixed.
Then there is a second support ranges where it was available (version not known) and prefixed. 
The rendered tables take the first array element as the one to render in the main cell. The other support ranges will be treated as secondary and put into the expandable info. See e.g. https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/imageSmoothingEnabled#Browser_compatibility